src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $cardWidth);
+  gap: 48px;
+
+  @each $width, $column in $pageStructure {
+    @media (min-width: $width) {
+      grid-template-columns: repeat($column, $cardWidth);
+    }
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cardWidth;
```

Comment:
But you set this width to grid column, why to repeat it here? Card should have same width as column, so it's enough to set it to column

Reply:
Fix it please

Reply:
done!:)